gems/aws-sdk-core/lib/aws-sdk-core/client_stubs.rb

```diff
@@ -166,6 +182,20 @@ def stub_responses(operation_name, *stubs)
       end
     end
 
+    # Allows you to access all of the requests that the stubbed client has made
+    #
+    # @return [Array] Returns an array of the api requests made
```

Comment:
If not create a delicate class for it, could you document that each element in the array is actually a hash containing request info? And supported keys for those hashes?

Reply:
I added documentation for everything inside of the log hash, but since the params hash can be variable based on whatever operation the client made I don't think enumerating the supported keys for it is feasible